string name                 # Primitive's name
--- 
